scripts/docker_deploy.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
+version="0.4.10"
```

Comment:
The version shouldn't be hardcoded here. Check how the other scripts determine the version.